internal/daemonset/daemonset.go

```diff
@@ -92,6 +103,10 @@ func (dc *daemonSetGenerator) SetDriverContainerAsDesired(
 		dc.kernelLabel:            kernelVersion,
 		constants.DaemonSetRole:   moduleLoaderRoleLabelValue,
 	}
+	if mld.ModuleVersion != "" {
```

Comment:
Can we move this logic to https://github.com/kubernetes-sigs/kernel-module-management/blob/bc488304f926742c14b0837563ef63acfef2b158/internal/module/kernelmapper.go#L118? I think that the namespace should be set to `default` instead of ` ` as soon as possible in the code.

Reply:
Setting the Version to default 1, will eventually require us or user to create a label on the nodes. This way , customers who don't want/need  to use ordered upgrade functionality can entirely skip it